src/crashsim.py

```diff
@@ -18,6 +18,7 @@
 import subprocess
 import logging
 from rrtest import create_test, configure_test, list_test, pack_test, analyze_test 
+from mutate import mutate
```

Comment:
This is a small concern, but the name mutate is weird here given that we aren't mutating.  We're identifying.

Reply:
Done